ratpack-core/src/test/groovy/ratpack/http/client/HttpClientIdleTimeoutSpec.groovy

```diff
@@ -116,6 +127,7 @@ class HttpClientIdleTimeoutSpec extends BaseHttpClientSpec {
 
     @Override
     void error(Context context, Throwable throwable) throws Exception {
+      throwable.printStackTrace()
```

Comment:
remove

Reply:
Whoops.

Reply:
@johnrengelman if you add another test where you sleep > timeout in between requests, you can assert that the channel is closed/null? and not in the pool.

Reply:
That's what the 2nd test case in `test client with idle timeout exceeded()`  tests. the `reused=false` condition on that test.